plugin/nvimgdb.vim

```diff
@@ -9,13 +9,17 @@ function! s:Spawn(backend, proxy_cmd, client_cmd)
   call luaeval("require'nvimgdb'.new(_A[1], _A[2], _A[3])", [a:backend, a:proxy_cmd, cmd])
 endfunction
 
-command! -nargs=1 -complete=shellcmd GdbStart call s:Spawn('gdb', 'gdb_wrap.sh', <q-args>)
+command! -nargs=1 -complete=custom,ExecsCompletion GdbStart call s:Spawn('gdb', 'gdb_wrap.sh', <q-args>)
 command! -nargs=1 -complete=shellcmd GdbStartLLDB call s:Spawn('lldb', 'lldb_wrap.sh', <q-args>)
```

Comment:
I believe lldb can also benefit from the cmake completion. Do you agree?

Reply:
I'm unfamiliar with lldb. I could just add this completion function to it though